.github/workflows/openapi.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
 
       - name: Validate open-api-spec.yaml
         run: |
+          echo "{\"extends\":[\"spectral:oas\",\"spectral:asyncapi\"]}" >> .spectral.json
```

Comment:
Why is that needed?

Reply:
It's required by the validator.
I removed the file that we had in the root of the repo in 3afb26ede2ac6c5c943ae712baf5198e914d3be9